iml-agent/src/cli.rs

```diff
@@ -53,8 +53,10 @@ fn parse_duration(src: &str) -> Result<u32, io::Error> {
         .map_err(|_| invalid_input_err(&format!("Could not parse {} to u32", val)))?;
```

Comment:
Parse here needs updating to u64 as well.

Reply:
Thanks, I meant to include this in the same commit and forgot to add it.